source/frontend/src/features/properties/list/columns.tsx

```diff
@@ -73,4 +79,47 @@ export const columns = ({ municipalities }: Props): ColumnWithProps<IProperty>[]
       },
     },
   },
+  {
+    Header: '',
+    accessor: 'controls' as any, // this column is not part of the data model
+    align: 'right',
+    sortable: false,
+    width: 20,
+    Cell: (cellProps: CellProps<IProperty, number>) => {
+      const { hasClaim } = useKeycloakWrapper();
+
+      return (
+        <StyledDiv>
+          {hasClaim(Claims.PROPERTY_VIEW) && (
+            <StyledIconButton
+              data-testid="view-prop-tab"
+              title="View Property Info"
+              variant="light"
+            >
+              <Link to={`/lease/${cellProps.row.original.id}`}>
+                <FaEye />
+              </Link>
+            </StyledIconButton>
+          )}
+
+          {hasClaim(Claims.PROPERTY_VIEW) && (
+            <StyledIconButton data-testid="view-prop-ext" title="View Property" variant="light">
+              <Link
+                to={`/lease/${cellProps.row.original.id}`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <FaExternalLinkAlt />
+              </Link>
+            </StyledIconButton>
```

Comment:
Are we sure these are the right links? Older code had different urls...

![image](https://user-images.githubusercontent.com/24322224/200077508-7d909d94-ef78-42dc-a6b2-759c4afea45d.png)

Reply:
my bad got confused with other story, this is corrected now. Thank you!